utils/odoo.go

```diff
@@ -55,14 +55,16 @@ func GetInstanceType() (string, error) {
 // FilterStrings receive a slice of strings and filter them by the 'odoorc_' prefix because these are the variables that
 // will be replaced in the configuration file, returns them as a map of strings where the key is the key of the
 // configuration
-func FilterStrings(list []string) map[string]string {
+func FilterStrings(list []string, filter bool) map[string]string {
 	res := make(map[string]string)
```

Comment:
@Angelfentanez instead of a boolean, pass a filter func, something like:

```go

type func envConverter([]string) map[string]string

func defaultConverter([]string) map[string]string {
    // just split, but don't  filter
    return splitEnvVars()
}

func odoorcConverter([]string) map[string]string {
    // filter and split
}

func splitEnvVars([]string) map[string]string {
    ...
    parts := strings.SplitN(v, "=", 2)
    if len(parts) < 2 {
        continue
    }
    ...
    return res
}

func FilterStrings(list []string, converter envConverter) map[string]string {
    ...
    list := converter(vars)
    for k, v := range list {
        key := strings.TrimPrefix(strings.ToLower(k), "odoorc_")
        res[key] =
     }
    ...
}


```

Reply:
@ruiztulio  Can you check this new version to see if that's what you had in mind? Though I'm not sure if we should remove the `FilterStrings` and `GetOdooVars` functions and call the converters directly if so.